src/main/resources/init_db.sql

```diff
@@ -18,3 +18,22 @@ CREATE TABLE `drivers` (
                                   PRIMARY KEY (`id`),
                                   UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
                                   UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+CREATE TABLE `cars` (
+                        `id` bigint NOT NULL AUTO_INCREMENT,
+                        `model` varchar(255) DEFAULT NULL,
+                        `is_deleted` tinyint NOT NULL DEFAULT '0',
+                        `manufacturer_id` bigint DEFAULT NULL,
+                        PRIMARY KEY (`id`),
+                        KEY `cars_manufacturers_fk` (`manufacturer_id`),
+                        CONSTRAINT `cars_manufacturers_fk` FOREIGN KEY (`manufacturer_id`) REFERENCES `manufacturers` (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=7 DEFAULT CHARSET=utf8mb3;
+
+CREATE TABLE `cars_drivers` (
+                                `driver_id` bigint NOT NULL,
+                                `car_id` bigint NOT NULL,
+                                `is_deleted` tinyint NOT NULL DEFAULT '0',
```

Comment:
remove this column, cars_drivers table is not an entity table, so we can just delete records from it, without soft delete